src/qt/forms/researcherwizardinvestorpage.ui

```diff
@@ -91,7 +91,7 @@
    <item alignment="Qt::AlignVCenter">
     <widget class="QLabel" name="investorLabel">
      <property name="text">
-      <string>Gridcoin will not participate in the research reward protocol. Your wallet will earn staking rewards only.</string>
+     <string>You have opted out of research rewards and will only be able to earn staking rewards.</string>
```

Comment:
```suggestion
     <string>You opted out of research rewards and will earn staking rewards only.</string>
```

Drops some passive voice/redundancy. My family is full of language arts nerds. Can't unsee it...

Reply:
Better.